proknow-sdk/Patient/Entities/StructureSetItem.cs

```diff
@@ -268,8 +268,8 @@ public void ReleaseLock()
                     foreach (var ex in ae.InnerExceptions)
                     {
                         // Handle the situation where the lock expired before we could delete it (e.g., when unit testing)
-                        if (!(ex is ProKnowHttpException) ||
-                            ex.Message != "HttpError(Forbidden, Structure set is not currently locked for editing)")
+                        if (!(ex is ProKnowHttpException) || (ex as ProKnowHttpException).ResponseStatusCode != "Forbidden" ||
+                            !ex.Message.Contains("Structure set is not currently locked for editing"))

```

Comment:
Does this line need a cast as well?

```suggestion
                        if (!(ex is ProKnowHttpException) || (ex as ProKnowHttpException).ResponseStatusCode != "Forbidden" ||
                            !(ex as ProKnowHttpException).Message.Contains("Structure set is not currently locked for editing"))
```

Reply:
It doesn't need a cast because ProKnowHttpException is derived from System.Exception which is where the Message property is defined.